### PR TITLE
Updated copy and styles of direct token swap

### PR DESCRIPTION
**Pull request by uiuxxx**

### CODE REVIEW

**Review by agualis — APPROVED** [review]

File: packages/lib/modules/pool/PoolDetail/PoolHeader/PoolAdvancedOptions.tsx
Comment (garethfuller) [inline packages/lib/modules/pool/PoolDetail/PoolHeader/PoolAdvancedOptions.tsx:1]: Can you remove the `PopoverCloseButton` import now that it's not used?

**Review by garethfuller — CHANGES_REQUESTED** [review]

File: packages/lib/modules/pool/PoolDetail/PoolHeader/PoolAdvancedOptions.tsx
Comment (agualis) [reply packages/lib/modules/pool/PoolDetail/PoolHeader/PoolAdvancedOptions.tsx:1]: I'll fix it.

File: packages/lib/modules/pool/PoolDetail/PoolHeader/PoolAdvancedOptions.tsx [hunk 1]
```diff
@@ -54,9 +52,9 @@ export function PoolAdvancedOptions() {
                   variants={staggeredFadeInUp}
                 >
                   <HStack>
-                    <SwapIcon size={24} />
+                    <SwapIcon size={20} />
                     <Link as={NextLink} href={`${pathname}/swap`} prefetch variant="nav">
-                      Swap tokens directly via this pool
+                      Force token swap via this pool
```
[the remark this answers is not in the document]
Comment (garethfuller) [reply packages/lib/modules/pool/PoolDetail/PoolHeader/PoolAdvancedOptions.tsx:57]: @agualis talked with Pon, can you update this please?

[the remark this answers is not in the document]
Comment (agualis) [reply packages/lib/modules/pool/PoolDetail/PoolHeader/PoolAdvancedOptions.tsx]: Done

**Review by garethfuller — APPROVED** [review]

**Pull request merged**